app/src/utils/chains.ts

```diff
@@ -144,22 +149,26 @@ const ALL_CHAIN_INFO: ChainInfo = {
     nativeCurrency: { name: 'Ethereum', symbol: 'ETH', decimals: 18 },
     grantRegistry: '',
     grantRoundManager: '',
+    grantRoundManagerAbi: GRANT_ROUND_MANAGER_ABI_UNI_V3_ABI,
     multicall: '0x5BA1e12693Dc8F9c48aAD8770482f4739bEeD696',
     rpcUrl: `https://eth-mainnet.alchemyapi.io/v2/${ALCHEMY_API_KEY}`,
+    startBlock: 13285176,
   },
   [SupportedChainId.POLYGON]: {
-    bridge: 'https://bridge.arbitrum.io/',
-    explorer: 'https://arbiscan.io/',
-    label: 'Arbitrum',
-    logoUrl: '/arbitrum_logo.svg',
+    bridge: 'https://wallet.polygon.technology/bridge/',
```

Comment:
do we remove the trailing `/` ?

Reply:
Not necessary since, unlike the etherscan URLs, we don't modify this (we don't even use it currently)